code/crates/network/src/lib.rs

```diff
@@ -291,6 +289,11 @@ async fn run(
         };
     }
 
+    // Timer to periodically try reconnecting to persistent peers
+    // TODO: Using 1 second for now, for faster reconnection during testing
+    // Maybe adjust via config in the future
+    let mut persistent_peer_timer = tokio::time::interval(std::time::Duration::from_secs(1));
+
```

Comment:
Any comments on this? 
The actual retry intervals given the fibonacci retry on connection failure will be:
1, 1, 2, 3, 5, 8, 1, 1, 2, 3, 5, 8,...
@cason @nenadmilosevic95 @romac ??

Reply:
This is complex, in any case, since we are considering only the sleep time of the Fibonacci, not the time to a connection attempt to fail, which may vary a lot. We want here to pick guys to whom attempts have failed but are set as persistent.

Reply:
Let's leave it 1sec for now. We can revise later.